src/main/scala/goatrodeo/Main.scala

```diff
@@ -68,7 +68,7 @@ object Howdy {
     import builder._
     OParser.sequence(
       programName("goatrodeo"),
-      head("goatrodeo", "0.4.6"),
+      head("goatrodeo", "0.5.0-SNAPSHOT"),
```

Comment:
We could use sbt-buildinfo here to automatically set this to the same value as [`version` in *build.sbt*](https://github.com/spice-labs-inc/goatrodeo/pull/29/files#diff-5634c415cd8c8504fdb973a3ed092300b43c4b8fc1e184f7249eb29a55511f91L15).

Reply:
Yay! That'd be a great enhancement!